joplin/base/migrations/0032_servicepage_steps.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 2.0.8 on 2018-09-27 04:54
```

Comment:
Here we begin the migration process, our first step is to add the new field without deleting the old steps relation. This lets us do a data migration from one to the other.